deploy-deps.sh

```diff
@@ -3,12 +3,48 @@
 script_path="$(dirname -- "${BASH_SOURCE[0]}")"
 
 main() {
+    echo "Checking requirements" >&2
+    check_req
     echo "Deploying Konflux Dependencies" >&2
     deploy
     echo "Waiting for the dependencies to be ready" >&2
     "${script_path}/wait-for-all.sh"
 }
 
+check_req(){
+    # declare the requirements
+    requirements=(kubectl openssl)
+    uninstalled_requirements=()
+
+    # check if requirements are installed
+    for i in ${requirements[@]}
+    do
```

Comment:
can we have the `do` in the same line with the for? (I wish we had a formmater for those things).

Reply:
Done